src/main/java/frc/robot/subsystems/ClimberSubsystem.java

```diff
@@ -0,0 +1,87 @@
+package frc.robot.subsystems;
+
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+
+import frc.robot.config.Config;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class ClimberSubsystem extends SubsystemBase {
+
+    private WPI_TalonSRX m_climber;
+
+    private ClimberSubsystem() {
+
+        // Initialize the subsystem if the shooter exists
+        if (Config.CLIMBER_TALON != -1) {
+            initializeSubsystem();
+        }
+    }
+
+    /**
+     * Initialization process for the shooter to be run on robots with this
+     * mechanism.
+     */
+    private void initializeSubsystem() {
+        m_climber = new WPI_TalonSRX(Config.CLIMBER_TALON);;
+
+        m_climber.setInverted(true);
+    }
+
+
+    /**
+     * Run the climb motor ensuring that it is positive
+     */
+    public void climb() {
+        m_climber.set(1.0);
```

Comment:
I don't think you can directly call set() like this.
Example for percent set: m_climber.set(ControlMode.PercentOutput, 0.05);
We will test to determine the final number.